src/Form/FormAnalyzer.js

```diff
@@ -272,7 +272,7 @@ class FormAnalyzer {
             if (likelyASubmit) {
                 this.form.querySelectorAll('input[type=submit], button[type=submit]').forEach((submit) => {
                     // If there is another element marked as submit and this is not, flip back to false
-                    if (el.type !== 'submit' && el !== submit) {
+                    if (el.getAttribute('type') !== 'submit' && el !== submit) {
```

Comment:
I think this is what we want to do here - check if this element if it's explicitly a submit type, although I am not sure. There can be submit buttons that don't have the `submit` attribute, but this check is very specific, in case there are other buttons that are marked with `type=submit`.

Reply:
Lol, this is wild. Nice find.